modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/user/profile/mgt/SharedUserProfileClaimMgtTestCase.java

```diff
@@ -462,13 +805,156 @@ private LocalClaimReq buildLocalClaimReq(String claimURI, String displayName, St
+    }
+
+    private ApplicationResponseModel addApplication(String appName) throws Exception {
+
+        ApplicationModel application = new ApplicationModel();
+
+        List<String> grantTypes = new ArrayList<>();
+        Collections.addAll(grantTypes, "authorization_code", "implicit", "password", "client_credentials",
+                "refresh_token", "organization_switch");
+
+        List<String> callBackUrls = new ArrayList<>();
+        Collections.addAll(callBackUrls, OAuth2Constant.CALLBACK_URL);
+
+        OpenIDConnectConfiguration oidcConfig = new OpenIDConnectConfiguration();
+        oidcConfig.setGrantTypes(grantTypes);
+        oidcConfig.setCallbackURLs(callBackUrls);
+
+        InboundProtocols inboundProtocolsConfig = new InboundProtocols();
+        inboundProtocolsConfig.setOidc(oidcConfig);
+
+        application.setInboundProtocolConfiguration(inboundProtocolsConfig);
+        application.setName(appName);
+        application.setIsManagementApp(true);
+
+        application.setClaimConfiguration(setApplicationClaimConfig());
+        String appId = addApplication(application);
+
+        return getApplication(appId);
+    }
+
+    private HttpResponse sendTokenRequestForPasswordGrant(String username, String password,
+                                                          List<String> requestedScopes, String clientId,
+                                                          String clientSecret) throws Exception {
+
+        List<NameValuePair> parameters = new ArrayList<>();
+        parameters.add(new BasicNameValuePair("grant_type", OAuth2Constant.OAUTH2_GRANT_TYPE_RESOURCE_OWNER));
+        parameters.add(new BasicNameValuePair("username", username));
+        parameters.add(new BasicNameValuePair("password", password));
+
+        String scopes = String.join(" ", requestedScopes);
+        parameters.add(new BasicNameValuePair("scope", scopes));
+
+        List<Header> headers = new ArrayList<>();
+        headers.add(new BasicHeader(AUTHORIZATION_HEADER,
+                OAuth2Constant.BASIC_HEADER + " " + getBase64EncodedString(clientId, clientSecret)));
+        headers.add(new BasicHeader("Content-Type", "application/x-www-form-urlencoded"));
+        headers.add(new BasicHeader("User-Agent", OAuth2Constant.USER_AGENT));
+
+        return sendPostRequest(httpClient, headers, parameters,
+                getTenantQualifiedURL(ACCESS_TOKEN_ENDPOINT, tenantInfo.getDomain()));
+    }
+
+    private HttpResponse getOrganizationSwitchToken(String clientId, String clientSecret, String currentToken,
+                                                   String switchingOrganizationId, List<String> requestedScopes)
+            throws Exception {
+
+        List<NameValuePair> parameters = new ArrayList<>();
+        parameters.add(new BasicNameValuePair(OAuth2Constant.GRANT_TYPE_NAME, "organization_switch"));
+        parameters.add(new BasicNameValuePair("token", currentToken));
```

Comment:
shall we use the constants defined in OAuth2Constant class?

Reply:
Fixed in [3b01ec0](https://github.com/wso2/product-is/pull/23064/commits/3b01ec03b91f4889e6dce7f7147274ce5ca3da6c)